README.md

```diff
@@ -49,13 +49,11 @@ $ nimble install owlkettle
 
 ## Documentation
 
-You can find a reference of all widgets in [docs/widgets.md](docs/widgets.md).
+[Click here]() for reference documentation of all widgets, including libadwaita widgets. 
 
-A tutorial for building a simple Todo application can be found [here](docs/tutorial.md).
+[Click here]() for a tutorial for building a simple Todo application.
 
-Additional examples can be found in the [examples](examples) folder.
-
-A reference of all supported libadwaita widgets can be found in [docs/widgets_adwaita.md](docs/widgets_adwaita.md).
+[Click here]() for additional examples Additional examples can be found in the [examples](examples) folder.
```

Comment:
I would not change the text in this section. Currently the links also don't work neither on the website nor in the GitHub README.

Reply:
They can't yet work because what I'd want to link to there would be pages on the nimibook docs once they're deployed on the owlkettle gihub pages. Those don't exist yet since it hasn't been deployed yet, so basically those can only be added in later, or I can start anticipating what the link will look like once its deployed, but then it still won't work yet before the PR is merged.

Reply:
Ah, ok.